saylua/models/user.py

```diff
@@ -59,11 +59,14 @@ class User(db.Model):
     bio = db.Column(db.Text(), default="Hello, world!") # FIXME: This should really have a limit.
 
     def __repr__(self):
-        return '<User %r>' % self.display_name
+        return '<User %r>' % self.name()
```

Comment:
I wasn't going to mention it, but the more I see it the more this bothers me-- turn that name() call into a class setter with:

```python
@property
def name(self):
    return self.display_name.display_name
```

So that you can just do `display_name.name`